packages/compiler-sfc/src/compileScript.ts

```diff
@@ -1172,8 +1172,8 @@ function walkObjectPattern(
         const type = isDefineCall
           ? BindingTypes.SETUP_CONST
           : isConst
-          ? BindingTypes.SETUP_MAYBE_REF
-          : BindingTypes.SETUP_LET
+            ? BindingTypes.SETUP_MAYBE_REF
+            : BindingTypes.SETUP_LET
```

Comment:
There seem to be a lot of indentation changes in this PR. These changes have been present on `main` for a long time, so I'm unsure why they're showing up here. Perhaps you need to merge in the latest `main`?

Reply:
https://github.com/vuejs/core/pull/8988#discussion_r1566665087